zerver/tests/test_create_video_call.py

```diff
@@ -61,7 +62,32 @@ def test_create_video_request_success(self) -> None:
             json={"join_url": "example.com"},
         )
 
-        response = self.client_post("/json/calls/zoom/create")
+        response = self.client_post("/json/calls/zoom/create", {"is_video_call": "true"})
+        self.assertEqual(
+            responses.calls[-1].request.url,
+            "https://api.zoom.us/v2/users/me/meetings",
+        )
+        self.assertEqual(
+            responses.calls[-1].request.headers["Authorization"],
+            "Bearer newtoken",
+        )
```

Comment:
It would be nice if we checked that the API request to Zoom had the expected parameters given the `is_video_call` value.

Reply:
Yes, good feedback. Should have included it. Thanks.

See #26595.